src/store/modules/plex/actions.js

```diff
@@ -93,25 +177,26 @@ export default {
                   tempConnection[key] = connection[key];
                 }
                 tempConnectionsArray.push(tempConnection);
-                if (connection.local === '1' && connection.uri.indexOf('plex') > -1) {
+                if (connection.local === '1' && connection.uri.indexOf('plex') > -1 && connection.uri.indexOf('com') > -1) {
```

Comment:
```
[...] confirmed works with android (phone app and shield tv) assuming your plex domain has an ssl cert and has com in it...
```
Just putting this here for ease of reference and to make sure we don't forget 😃

Reply:
Setting it to need com or org in it. The ones we want to avoid are the direct ips and plex.direct

Reply:
I think something like this would help https://www.npmjs.com/package/parse-domain
Have it parse the URI, then check that tld isn't empty or `direct`